### PR TITLE
Modify theme for dismissable banner

### LINKED ISSUE #1580: Color change for Link in Banner for A11Y
### Currently
The link that's currently in the [dismissable banner](http://storybook.backstage.io/?path=/story/dismissablebanner--with-link) component doesn't pass A11Y standards. If we could change it to #000000, that would be great! Also the background color for the default banner should be #2E77D0. The background color for the error banner should be #E22134.

Screenshot 2020-07-10 at 10 50 46 AM

### Ideally
Screenshot 2020-07-10 at 10 51 54 AM

### PR DESCRIPTION
**Pull request by twishasaraiya**
## Hey, I just made a Pull Request!

Fixes #1580 

#### :heavy_check_mark: Checklist

- [x] All tests are passing `yarn test`
- [x] Screenshots attached (for UI changes)
- [ ] Relevant documentation updated
- [x] Prettier run on changed files
- [ ] Tests added for new functionality
- [ ] Regression tests added for bug fixes

**Banner Changes**
Screenshot from 2020-07-13 21-11-43

**Quick observations**
-  When I did a quick search, `primary.main` the blue colour was used only for the dismissable banner. So it should not affect any other component

### CODE REVIEW

**Comment by benjdlambert** [comment]
>When I did a quick search, primary.main the blue colour was used only for the dismissable banner. 

I think that it's used for a little more than that, the CTA button has also changed too. Think you might have to do this a different way otherwise it will affect those buttons too.

**Comment by twishasaraiya** [comment]
Okay, so what I am planning to do is 

1. And add custom properties to `PaletteAdditions` in `types.ts`

```
banner: {
    info: string;
    error: string;
    link: {
      default: string;
    };
  }
```

2. Set the default values in `themes.ts`

```js
banner: {
      info: '#2E77D0',
      error: '#E22134',
      link: {
        default: '#000000',
      },
    }
```

3. Change `Theme` to `BackstageTheme` in `DismissableBanner.tsx`

```js
const useStyles = makeStyles((theme: BackstageTheme) => ({
```

Let me if this is the right approach to go-ahead with.

**Comment by twishasaraiya** [comment]
Screenshot from 2020-07-15 12-31-46

My screen resolution is not so good. so sharing these as well

Screenshot from 2020-07-15 12-33-28
Screenshot from 2020-07-15 12-33-24


Let me know if some changes need to be made. I am actually new to React

**Comment by twishasaraiya** [comment]
Hi @benjdlambert, I made some changes last week, if you could please review it and let me know if anything else is required.

File: packages/theme/src/themes.ts [hunk 1]
```diff
@@ -90,6 +94,10 @@ export const darkTheme = createTheme({
     primary: {
       main: blue[500],
     },
+    banner: {
+      info: '2E77D0',
```
Comment (benjdlambert) [inline packages/theme/src/themes.ts:98]: Why are these needed?

They don't have a `#` in front so I guess they're not gonna work anyway.

**Review by benjdlambert — APPROVED** [review]

**Comment by benjdlambert** [comment]
perfect! thanks!
